src/main/java/org/anystub/jdbc/ResultSetUtil.java

```diff
@@ -162,18 +170,18 @@ private static String encodeValue(ResultSet resultSet, int columnType, int colum
 //            case  BIT = -7;
                 case TINYINT:
                 case SMALLINT:
-                    return String.valueOf(resultSet.getShort(column));
+                    return NUMBER_CONVERTER.format(resultSet.getShort(column));
```

Comment:
I wonder if it is possible to implement with fasterxml?
Could you try it with ObjectMapper which you can take from ObjectMapperFactory

Reply:
Sorry, I haven't enough time for such experiments as of now. I found solution that works. And this is not emergency. At least for me since nowadays I am using the fork with all 3 fixes I've made. These PRs was simply my attemt to contribute this to upstream. So if you think it worth to find better approach, you may just close this PR. I may return to the problem later, when I have some spare time.